includes/admin/class-wp-job-manager-admin.php

```diff
@@ -128,6 +125,9 @@ public function admin_enqueue_scripts() {
 
 		WP_Job_Manager::register_style( 'job_manager_admin_menu_css', 'css/menu.css', [] );
 		wp_enqueue_style( 'job_manager_admin_menu_css' );
+
+		WP_Job_Manager::register_style( 'job_manager_admin_notices_css', 'css/admin-notices.css', [] );
+		wp_enqueue_style( 'job_manager_admin_notices_css' );
```

Comment:
I've moved this outside of the conditional above as I was adding so many different screen IDs that the check was becoming pointless. It's also not possible to anticipate IDs as users will have very many different screens based on very many different plugins, and loading the styles globally makes the most sense to me.

Reply:
It makes sense to me too! Just to confirm, I noticed you also removed the `dashboard` from the conditional. Was it there only for the notices?

Reply:
That's right. I added that here https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/50a8a711471ebed61d76c9222f458c1524530dce.